internal/namespace/manager.go

```diff
@@ -124,3 +139,129 @@ func (nm *namespaceManager) initNamespaces(ctx context.Context, di database.Plug
 	}
 	return nil
 }
+
+func (nm *namespaceManager) validateNamespaceConfig(ctx context.Context, name string, index int, conf config.Section) error {
+	if err := core.ValidateFFNameField(ctx, name, fmt.Sprintf("namespaces.predefined[%d].name", index)); err != nil {
+		return err
+	}
+
+	if strings.ToLower(name) == "ff_system" || strings.ToLower(conf.GetString(coreconfig.NamespaceRemoteName)) == "ff_system" {
```

Comment:
Can use the `core.SystemNamespace` constant here

Reply:
Might want to parameterize the error message as well, ie `%s is a reserved name` and then you can pass in `core.SystemNamespace`